src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -956,8 +946,14 @@ async function main(): Promise<void> {
   }
 
   InitializeEnvironmentalVariables(frb);
-  if (isFeatureEnabled(frb, "protect_env")) {
```

Comment:
This is the behavior change.

Reply:
@samtstern looks nice! the console output is awesome!